src/test/java/org/openrewrite/java/migrate/UpgradeToJava6Test.java

```diff
@@ -86,52 +92,57 @@ public int getLoginTimeout() throws SQLException {
+              			throws SQLException {
+              		// TODO Auto-generated method stub
+              		return null;
+              	}
                                
-                public Connection getConnection(String username, String password)
-                    throws SQLException {
-                  // TODO Auto-generated method stub
-                  return null;
-                }
+              	public PrintWriter getLogWriter() throws SQLException {
+              		// TODO Auto-generated method stub
+              		return null;
+              	}
                                
-                public PrintWriter getLogWriter() throws SQLException {
-                  // TODO Auto-generated method stub
-                  return null;
-                }
+              	public void setLogWriter(PrintWriter out) throws SQLException {
+              		// TODO Auto-generated method stub
                                
-                public void setLogWriter(PrintWriter out) throws SQLException {
-                  // TODO Auto-generated method stub
-                }
-                
-                public void setLoginTimeout(int seconds) throws SQLException {
-                  // TODO Auto-generated method stub
-                }
+              	}
+                               
+              	public void setLoginTimeout(int seconds) throws SQLException {
+              		// TODO Auto-generated method stub
+                               
+              	}
+                               
+              	public int getLoginTimeout() throws SQLException {
+              		// TODO Auto-generated method stub
+              		return 0;
+              	}
                                
-                public int getLoginTimeout() throws SQLException {
-                  // TODO Auto-generated method stub
-                  return 0;
-                }
-                            
                   public boolean isWrapperFor(Class<?> iface) throws java.sql.SQLException {
+                      // TODO Auto-generated method stub
                       return iface != null && iface.isAssignableFrom(this.getClass());
                   }
-               
+                               
                   public <T> T unwrap(Class<T> iface) throws java.sql.SQLException {
+                      // TODO Auto-generated method stub
                       try {
-                          if (isWrapperFor(iface)) {
+                          if (iface != null && iface.isAssignableFrom(this.getClass())) {
                               return (T) this;
                           }
-                          throw DbException.getInvalidValueException("iface", iface);
+                          throw new java.lang.Exception();
                       } catch (Exception e) {
-                          throw logAndConvert(e);
+                          throw new java.sql.SQLException();
```

Comment:
Likely best to include the cause here to aid debugging, right?
```suggestion
                          throw new java.sql.SQLException(e);
```